frontend/www/js_templates/l2/endpoint_selection_modal.js

```diff
@@ -311,7 +311,7 @@ class EndpointSelectionModal2 {
     // describes a cloud interconnect, disable the Interfaces as they
     // will be auto-selected according to the prescribed procedure.
     for (let i = 0; i < entity.interfaces.length; i++) {
-      let child = entity.interfaces[i];
+      var child = entity.interfaces[i];
```

Comment:
Unless there's a reason for using `var`, `let` is preferred as it limits the variables scope.